libs/blocks/merch-card/merch-card.js

```diff
@@ -248,12 +266,6 @@ const parseContent = async (el, merchCard) => {
     if (mnemonicList) bodySlot.append(mnemonicList);
   });
 
-  if (calloutContainer.children.length > 0) {
-    const calloutSlot = createTag('div', { slot: 'callout-content' });
-    calloutSlot.appendChild(calloutContainer);
-    merchCard.appendChild(calloutSlot);
```

Comment:
@rohitsahu does the order of adding the callout slot matter

Reply:
Thanks guys for bringing my attention to this, I rewrote the code, not all works as it should